scripts/apps/web-publisher/services/PubAPIFactory.js

```diff
@@ -52,6 +64,23 @@ export function PubAPIFactory(config, $http, $q) {
 
         /**
          * @ngdoc method
+         * @name pubapi#get
+         * @param {String} resource
+         * @param {Number} id
+         * @returns {Promise}
+         * @description GET a given resource by id.
+         */
+        get(resource, id) {
+            return this.req({
+                url: this.resourceURL(resource, id),
+                method: 'GET'
+            }).then(response => {
```

Comment:
This `.then` clause - why is it needed? Doesn't the promise already return it's response?